@xen-orchestra/web/src/remote-resources/use-xo-user-collections.ts

```diff
@@ -0,0 +1,14 @@
+import { useXoCollectionState } from '@/composables/xo-collection-state/use-xo-collection-state.ts'
+import { defineRemoteResource } from '@core/packages/remote-resource/define-remote-resource.ts'
+import type { XoUser } from '@vates/types'
+
+export const useXoUserCollection = defineRemoteResource({
```

Comment:
Shouldn't the collection be reactive?
If yes, you should have to extract the fields in a variable and use the `watchCollection` option, like other reactive collections.